pkg/endpoint/endpoint.go

```diff
@@ -128,6 +128,11 @@ type Endpoint struct {
 	// recalculated on endpoint restore.
 	createdAt time.Time
 
+	// lastUpdatedTime stores the time the endpoint was last changed.
+	// It is used to calculate the endpoint propagation delay.
+	// We use a map, rather than a single value, to account for concurrent operations.
```

Comment:
nit: please add what is key and what is value in the comment.

Reply:
done it.:)